GliaWidgets/View/Call/CallStyle.swift

```diff
@@ -1,23 +1,80 @@
 import UIKit
 
+/// Style of the call view.
 public class CallStyle: EngagementStyle {
+    /// Title for audio call.
     public var audioTitle: String
+
+    /// Title for video call.
     public var videoTitle: String
+
+    /// Back button style in the header.
     public var backButton: HeaderButtonStyle
+
+    /// Close button style in the header.
     public var closeButton: HeaderButtonStyle
+
+    /// A text to display operator's name. Include `{operatorName}` template parameter in the string to display operator's name.
     public var operatorName: String
+
+    /// Font of the operator's name text.
     public var operatorNameFont: UIFont
+
+    /// Color of the operator's name text.
     public var operatorNameColor: UIColor
+
+    /// Font of the call's duration text.
     public var durationFont: UIFont
+
+    /// Color of the call's duration text.
```

Comment:
Again, `'s` is not needed. This is the duration of a call, not a duration that belongs to a call.